homeassistant/components/samsungtv/media_player.py

```diff
@@ -256,13 +256,13 @@ def _get_upnp_service(self, log: bool = False) -> UpnpService | None:
                 LOGGER.info("Upnp services are not available on %s", self._host)
             return None
 
-        if service := self._upnp_device.services.get(UPNP_SVC_RENDERINGCONTROL):
```

Comment:
I didn't add support for the new service as I didn't want to make this any larger